### PR TITLE
Add logo to readme

### LINKED ISSUE #4005: mermaid logo proposal
### Proposal

This seems a proper place to offer this image as repository proposal. See Screenshots

### Colors

RGB [0.933, 0.51, 0.933]

### Screenshots

https://imgur.com/a/XC0gZ3q

### PR DESCRIPTION
**Pull request by sidharthv96**
## :bookmark_tabs: Summary

Adds mermaid logo to the readme.
Inspired from Vitest's readme. 

Resolves #4005 

## Old
image

## New
image

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [x] :notebook: have added documentation (if appropriate)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

**Review by aloisklink — APPROVED** [review]
Looks really nice!

Btw, is `packages/mermaid/README.md` supposed to be exactly the same as `README.md`?

If so, maybe we should look into some way of ensuring they stay in sync. Maybe using symlinks (assuming that works on Windows).

File: README.md [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+<img src="https://raw.githubusercontent.com/mermaid-js/mermaid/develop/docs/public/favicon.svg" height="150">
+</p>
+<h1 align="center">
+Mermaid
+</h1>
+<p align="center">
+Generate diagrams from markdown-like text.
+<p>
+<p align="center">
+  <a href="https://www.npmjs.com/package/vitest"><img src="https://img.shields.io/npm/v/mermaid?color=ff3670&label="></a>
```
[the remark this answers is not in the document]
Comment (sidharthv96) [reply README.md:11]: I did try that, the clean look of having just `v9.3.0` was hard to beat.

**Comment by sidharthv96** [comment]
> Btw, is packages/mermaid/README.md supposed to be exactly the same as README.md?
> 
> If so, maybe we should look into some way of ensuring they stay in sync. Maybe using symlinks (assuming that works on Windows).

The readme inside mermaid is for the npm package page. We could use a prepublish hook to copy the main readme. I think I tried symlinks a while back, but it didn't work.

**Comment by sidharthv96** [comment]
image

@aloisklink 

- Duplicate readmes removed. Will be copied before publish.
- CDN link moved to top, changelog and rest deleted.
- Icons added to top links.

**Review by aloisklink — APPROVED** [review]
> I think I tried symlinks a while back, but it didn't work.

Damn, that sucks. Copying should work just as well.

> Icons added to top links.

Awesome, I do feel like it does look slightly nicer with them.

Also, you're a hero for splitting up the badges into separate lines, it should make future PRs so much easier to review! :rocket:  https://github.com/mermaid-js/mermaid/blob/3e9978a58c5189ed83fc3486bff098f15a4e3662/README.md?plain=1#L27-L34

**Pull request merged**